brigade/views.py

```diff
@@ -296,8 +303,91 @@ def project_monitor(brigadeid=None):
     return render_template('monitor.html', projects=projects_with_tests, org_name=brigadeid)
 
 
-@redirect_from('/brigade/', '/brigade/list')
+@redirect_from('/brigade/list', '/brigades/')
```

Comment:
The routing here is weird. I'm actually kind of surprised it works to redirect from the URL with the trailing slash to the URL without. Did you get a chance to test that?